ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -522,13 +523,13 @@ tests:
       IP_STACK: v4
```

Comment:
not sure why ztp-periodic is missing the dependencies stanza. I think it's not intended